flightros/src/racing/racing.cpp

```diff
@@ -69,8 +69,8 @@ int main(int argc, char *argv[]) {
   std::size_t num_waypoints = way_points.size();
   Eigen::VectorXd segment_times(num_waypoints);
   segment_times << 10.0, 10.0, 10.0, 10.0;
-  Eigen::VectorXd minimization_weights(num_waypoints);
-  minimization_weights << 1.0, 1.0, 1.0, 1.0;
+  Eigen::VectorXd minimization_weights(5);
+  minimization_weights << 0.0, 1.0, 1.0, 1.0, 1.0;
```

Comment:
good catch! Weird that it works with melodic and that it causes an error in noetic. Does something change when the first weight is 1.0?

Reply:
It produces the same Eigen assertion if the first value is 1.0.

Reply:
@kelia any clue why that happens?

Reply:
see my answer below :+1: